source/shared/xplat.h

```diff
@@ -234,162 +117,48 @@ typedef windowsULongLong_t ULONGLONG;
 // source code shared with Windows SNAC (which is ODBC 3.8) compiles.
 #define ODBCVER 0x0380
```

Comment:
its not about appveyor. PECL build does not use --with-odbcver configure option.